.eslintrc

```diff
@@ -1,25 +1,29 @@
 {
+    "extends": "eslint:recommended",
     "parser": "babel-eslint",
     "plugins": ["babel"],
     "rules": {
-        "strict": 0,
-        "indent": [2, 4],
-        "quotes": [2, "single"],
-        "linebreak-style": [2, "unix"],
+        "consistent-return": 2,
```

Comment:
This is new. It disallows things like

```javascript
function() {
    if (foo) {
        return 2;
    } else {
        return;
    }
}
```

ie: returning a value in some places and nothing in other places.